packages/perseus/src/util/graphie.test.ts

```diff
@@ -40,7 +40,7 @@ function createFakeRaphaelSet() {
 }
 
 function createAndInitGraphie(): Graphie {
-    const graphie = GraphUtils.createGraphie();
+    const graphie = GraphUtils.createGraphie(document.createElement("div"));
```

Comment:
**thought**: Not sure if this is premature optimization, but it seems off that we don't add this div to the document.  Maybe the fact that it isn't needed is a statement that we don't assert anything about what Graphie does with this div yet?

Reply:
This is a cheap way of getting typechecking to pass. We apparently don't use this element in the code under test, since the tests were passing without it, but I figured that creating a dummy div is better than suppressing typechecking for this line.